input_dictionary.json

```diff
@@ -461,6 +461,120 @@
+          "type": "number",
+          "display_label": "Error weight",
+          "placeholder": "Insert parameter",
+          "tooltip": "Hamming weight of the target solution",
+          "dependencies": [
+            {
+              "id": "n",
+              "action": "validateLessThan"
+            },
+            {
+              "action": "validateLessThanOperation",
+              "operation": "n - k"
+            }
+          ]
+        },
+        {
+          "id": "q",
+          "type": "number",
+          "display_label": "Base Field Size",
+          "placeholder": "Insert parameter",
+          "tooltip": "Size of the underlying base field",
+          "dependencies": []
+        }
+      ],
+      "estimator_parameters": [
+        {
+          "id": "bit_complexities",
+          "type": "switch",
+          "display_label": "Bit complexities",
+          "default_value": true,
+          "tooltip": "Show complexities as count of bit operations. If false, show number of field additions"
+        },
+        {
+          "id": "included_algorithms",
+          "type": "multiple_selector",
+          "direction": "column",
+          "display_label": "Included algorithms",
+          "tooltip": "Algorithms to include for optimization",
+          "default_value": [],
+          "excluded_algorithms": [],
+          "options": [],
+          "dependencies": []
+        },
+        {
+          "id": "memory_bound",
+          "type": "number",
+          "direction": "column",
+          "display_label": "Memory limit",
+          "default_value": null,
+          "placeholder": "Insert value",
+          "caption": "Leave empty if no limit is desired",
+          "tooltip": "Log2 of the maximum number of bits of memory available"
+        },
+        {
+          "id": "memory_access",
+          "type": "selector",
+          "direction": "column",
+          "display_label": "Memory access cost",
+          "default_value": "Constant",
+          "tooltip": "Function that takes as input the memory bit complexity and outputs the associate algorithmic cost. Example, logarithmic memory access, input M, output M+log2M.",
+          "options": ["Constant", "Logaritmic", "Square root", "Cube root"]
+        },
+        {
+          "id": "precision",
+          "type": "number",
+          "direction": "column",
+          "display_label": "Decimal precision",
+          "default_value": 0,
+          "placeholder": "Insert value",
+          "tooltip": "Number of decimal digits to display"
+        },
+        {
+          "id": "nsolutions",
+          "type": "number",
+          "direction": "column",
+          "display_label": "Number of solutions",
+          "placeholder": "Insert value",
+          "tooltip": "Log2 of number of existing solutions of which one has to be found, if not specified it will be fixed to the number of solutions in expectation"
+        }
+      ],
+      "optional_parameters": [
+
```

Comment:
here should be the is_syndrome_zero parameter, named "Code word search" as a selector